pkg/controller/replicaset/replica_set.go

```diff
@@ -84,7 +92,7 @@ type ReplicaSetController struct {
 	// A store of pods, populated by the podController
 	podStore cache.StoreToPodLister
 	// Watches changes to all pods
-	podController *framework.Controller
+	podController framework.ControllerInterface
```

Comment:
Is this used for anything besides `HasSynced`?  If not, how about just pulling that directly from the informer?